curvefs/proto/metaserver.proto

```diff
@@ -315,8 +316,6 @@ service MetaServerService {
     rpc DeleteInode(DeleteInodeRequest) returns (DeleteInodeResponse);
     rpc CreateRootInode(CreateRootInodeRequest) returns
                                             (CreateRootInodeResponse);
-    rpc UpdateInodeS3Version(UpdateInodeS3VersionRequest) returns
```

Comment:
corresponding request and response definition should be deleted.

Reply:
fixed